src/ClimatePref/ReadData.jl

```diff
@@ -12,6 +12,7 @@ const AG = ArchGDAL
 
 vardict = Dict("bio" => NaN, "prec" => mm, "srad" => u"kJ"* u"m"^-2 * day^-1, "tavg" => K, "tmax" => K, "tmin" => K, "vapr" => u"kPa", "wind" => u"m" * u"s"^-1)
 unitdict = Dict("K" => K, "m" => m, "J m**-2" => J/m^2, "m**3 m**-3" => m^3, "kg m-2 s-1" => kg * m^-2 * s^-1, "1" => 1, nothing => W*m^-2, "d" => day)
+biodict = Dict(zip(1:19, [fill(K, 11); fill(kg/m^2, 8)]))
```

Comment:
Shouldn't these be `const` to avoid slowdowns in code that uses them? Also, if they are const global variables, they should be CAPITALISED throughout to make it obvious something weird is happening...

Reply:
These are in `main` though so maybe this needs to be put into a single commit there and cherry picked or something?